src-docs/src/views/key_pad_menu/key_pad_menu_example.js

```diff
@@ -84,11 +85,23 @@ export const KeyPadMenuExample = {
         },
       ],
       text: (
-        <p>
-          The <strong>EuiKeyPadMenuItem</strong> component can act both as an
-          anchor as well as a button by specifying <EuiCode>href</EuiCode> or
-          <EuiCode>onClick</EuiCode> respectively.
-        </p>
+        <>
+          <p>
+            The <strong>EuiKeyPadMenuItem</strong> component can act both as an
+            anchor as well as a button by specifying <EuiCode>href</EuiCode> or
+            <EuiCode>onClick</EuiCode> respectively.
+          </p>
+          <EuiCallOut iconType="accessibility" title="Accessibility pro tip">
+            <p>
+              If the menu is a bunch of links and is acting as navigation for
+              your app: wrap the whole thing in a{' '}
+              <EuiCode language="html">
+                &lt;nav aria-label=&quot;nav title&quot;&gt;&lt;/nav&gt;
+              </EuiCode>
+              .
+            </p>
+          </EuiCallOut>
```

Comment:
This can just be more precise and put the content as the title.

```suggestion
          <EuiCallOut 
            iconType="accessibility" 
            title={<>
              If the menu provides navigation for your application, wrap the <strong>EuiKeyPadMenu</strong> with{' '}
              <EuiCode language="html">
               {'<nav aria-label="Nav title"></nav>'}
              </EuiCode>.
            </>}
          />
```